### PR TITLE
Added Text Hierarchy in FAQ page

### PR DESCRIPTION
**Pull request by Tabhi109**
- Added `<h3>` tags for Questions.
- Added `<strong>` tags for "A:" (answers)
- Added `<strong>` tags for heading and "Layer5"

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b4d9711662397ad58da8579a1587926e821dcec0 at: https://64da58abf47cb20084496998--layer5.netlify.app

File: src/sections/Community/Handbook/faq.js [hunk 1]
```diff
@@ -17,43 +17,43 @@ const Faq = () => {
...
+            <p><strong>A:</strong> You can start by going through the <a href="https://layer5.io/community/newcomers">Newcomers’ Welcome Guide.</a></p>
 
-            <h4>Q: What exactly is a MeshMate?</h4>
-            <p>A:Layer5<Link to="/community/meshmates"> MeshMates </Link>are individuals committed to helping community members be successful contributors. Meshmates are there to guide and support the community members, helping them identify projects they can contribute to depending on their areas of interest, directing them on groups to join and helping them grow in their open-source and cloud native knowledge.</p>
+            <h3>Q: What exactly is a MeshMate?</h3>
+            <p><strong>A:</strong><strong>Layer5</strong><Link to="/community/meshmates"> MeshMates </Link>are individuals committed to helping community members be successful contributors. Meshmates are there to guide and support the community members, helping them identify projects they can contribute to depending on their areas of interest, directing them on groups to join and helping them grow in their open-source and cloud native knowledge.</p>
```
Comment (Mohith234) [inline src/sections/Community/Handbook/faq.js:27]: Add a space between `A:` and `Layer5`

File: src/sections/Community/Handbook/faq.js [hunk 2]
```diff
@@ -17,43 +17,43 @@ const Faq = () => {
         <Container>
           <div className="content">
             <h2>General FAQs:</h2><br />
-            <h4>Q: Are Layer5’s solutions open source?</h4>
-            <p>A: Yes, all Layer5 projects are licensed under Apache V2. Layer5 also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
+            <h3>Q: Are Layer5’s solutions open source?</h3>
+            <p><strong>A:</strong> Yes, all <strong>Layer5</strong> projects are licensed under Apache V2. <strong>Layer5</strong> also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
```
Comment (Mohith234) [inline src/sections/Community/Handbook/faq.js:21]: Also, I don't think making Layer5 bold creates an impact while looking at the FAQ page in the preview

- If you are really looking to change it into bold please do the same with all of the `Layer5` words on the page. Just to be consistent

**Review by Mohith234 — COMMENTED** [review]
- I've just suggested some changes, Feel free to add your opinion on it 

// @Tabhi109

File: src/sections/Community/Handbook/faq.js (hunk 2 above)
Comment (Tabhi109) [reply src/sections/Community/Handbook/faq.js:21]: Okay Sure , Thanks @Mohith234 !

**Comment by Tabhi109** [comment]
Will make changes and issue a new PR

**Pull request closed without merging**